### DISCUSSION TITLE
Multiple mutations in a single request with codegen plugin

### DISCUSSION THREAD

**Discussion by nimaaj93** [Netflix/dgs-framework#521] [General]
I am looking for a way to execute multiple mutations in a single Graphql request using the types and the clients generated by the codegen plugin. I see that `GraphQLQueryRequest` constructor only accepts one client object as an argument. 
Any way I could pass multiple mutation objects and get it serialised as a string?

**Comment by paulbakker** [comment]
This is currently not supported but would be a good addition. Feel free to create a feature request.

**Comment by manymotes** [comment]
https://github.com/Netflix/dgs-codegen/pull/426

**Reply by srinivasankavitha** [reply]
This should be available in the upcoming codegen release `v5.2.5`
